src/modules/hostname.rs

```diff
@@ -0,0 +1,39 @@
+use ansi_term::{Color, Style};
+use std::env;
+use std::process::Command;
+
+use super::{Context, Module};
+use std::ffi::OsString;
+
+/// Creates a module with the system hostname
+///
+/// Will display the hostname if all of the following criteria are met:
```

Comment:
Nice documenting. 👍

Reply:
Thanks, tried to match the style from other modules.